charts/bluehub/v0.1.0/questions.yaml

```diff
@@ -0,0 +1,329 @@
+labels:
+ io.rancher.certified: partner
+questions:
+- variable: zookeeper.enabled
```

Comment:
If Zookeeper, Kafka and Security Manager are required as a bare minimum, their values shouldn't be subquestions wrapped by a variable that enables them since users can choose to not install them and the installation will fail.

Please remove those from the enabled wrapper, and reorder question groups as following so required groups are at the top.

Zookeeper
Kafka
Security Manager
Schema Registry
Cluster Manager Settings
Kproducer

Reply:
The changes are committed as suggested